app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -33,12 +33,13 @@ def perform(_now)
     )
 
     started_at = Time.zone.now
+    enrollments.update(last_batch_claim_at: started_at)
+    enrollments = InPersonEnrollment.needs_usps_status_check_batch(started_at)
```

Comment:
I would consider naming this new scope something different, since it's a fresh query now. Maybe name them `previous_enrollments` and `enrollments_to_check`